Makefile

```diff
@@ -175,9 +189,100 @@ build/codegen:
 	  --doc_out=pkg/api/schema --doc_opt=markdown,docs.md \
 	  --plugin=protoc-gen-doc=$(PWD)/scripts/protoc-gen-doc
 
+.PHONY: build/codegendocuments
+build/codegendocuments:
+	$(PROTOC) -I pkg/api/documentsschema/ pkg/api/documentsschema/documentsschema.proto \
```

Comment:
probably `pkg/api/documentschema` is slightly more readable...

Reply:
Resolved to documentschema